test/ffi/booster.jl

```diff
@@ -286,27 +290,50 @@ end
 
 @testset "LGBM_BoosterSaveModel" begin
```

Comment:
nice, I would recommend cleaning up prior to running the test as well.

Reply:
9c03935 resolves